standards.md

```diff
@@ -315,4 +315,39 @@ The strings for each child key may contain a double colon (::) delimiter, which
 },
 ```
 
+## Governance
+
```

Comment:
do we need a "This is a living document" or Aims type statement here (or in the overall intro)?
Provide contact info for "an email must be sent to the EMDS team"— suggest a single address that can be forwarded as needed.

Reply:
I added a living document statement and a link to the governance section in the landing page: https://github.com/Earth-System-Diagnostics-Standards/EMDS/blob/add_governance_section/README.md#standards-document

Reply:
@dneelin for an email address, are you proposing adding emails for each of the committee members, or setting up some kind of EMDS alias?